CONNECTION_CLOSE is non-ack-eliciting (#3098)
quicwg/base-drafts: draft-ietf-quic-recovery.md

Ack-eliciting Frames:

> : All frames besides ACK, PADDING, or CONNECTION_CLOSE are considered

Reviewer: ```suggestion
: All frames other than ACK, PADDING, and CONNECTION_CLOSE are considered
```

  performance of the QUIC handshake and use shorter timers for
  acknowledgement.

> * Packets that contain only ACK or CONNECTION_CLOSE frames do not count toward

Reviewer 2: It's not clear if the "only" applies only to ACKs or to both.

Author: @ianswett Thank you for the suggestion. Applied in f7eed75f.
